README.md

```diff
@@ -1 +1,6 @@
-2020-jun-project1-group1
\ No newline at end of file
+2020-jun-project1-group1
+
+
+## Documents
+
+- [Docker Wordpress](docs/docker-wordpress.md)
```

Comment:
I think you can discard the changes in the README file. We can add all these references later.

Reply:
Done, thanks ;)